boxes.d/99-local.yaml.example

```diff
@@ -10,6 +10,25 @@ centos7-devel:
       foreman_devel_github_push_ssh: True
       katello_devel_github_username: <REPLACE ME>
 
+charon:
+  box: centos7
+  memory: 9000
+  ansible:
+    playbook: 'playbooks/devel.yml'
+    group: 'devel'
+    variables:
+      katello_devel_github_username: <REPLACE ME>
```

Comment:
I did consider using some system variable or ansible variable to automatically do this, but couldn't really find a good way. Closes was delegating one to localhost, register the ```ansible_username``` fact and defaulting to that later on.